holoviews/plotting/links.py

```diff
@@ -104,6 +104,12 @@ class RangeToolLink(Link):
     axes = param.ListSelector(default=['x'], objects=['x', 'y'], doc="""
         Which axes to link the tool to.""")
 
+    boundsx = param.Tuple(default=None, length=2, doc="""
+        (start, end) bounds for the x-axis""")
+
+    boundsy = param.Tuple(default=None, length=2, doc= """
```

Comment:
```suggestion
    boundsy = param.Tuple(default=None, length=2, doc="""
```

Very _very_ nit. Only do this if you make other changes...